lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        Create User
+    </title>
+</head>
+
+<body>
+    <h1>Create User</h1>
+    <hr>
+
+    <form action="create-user.html" method="GET">
+
+        <lable>Email:</lable>
+        <input type="email" name="email" placeholder="email" required />
+        <br>
+        <label> Username:</label>
+        <input type="text" name="username" placeholder="username" size="30" maxlength="12" />
+        <br>
+        <lable>Password:</lable>
+        <input type="password" name="password" placeholder=" password" size="30" maxlength="8" />
+        <br>
+        <label>bio:</label>
+        <br>
+        <textarea name="bio" placeholder="give a brief description of this user"></textarea>
+        <br>
```

Comment:
A better way to break these apart would be to surround each label + input group with a `<div>`, e.g.

```html
<div>
  <lable>Email:</lable>
  <input type="email" name="email" placeholder="email" required />
</div>

<div>
  <label> Username:</label>
  <input type="text" name="username" placeholder="username" size="30" maxlength="12" />
</div>

...etc
```